bin/logs.js

```diff
@@ -45,21 +45,24 @@ function handleLogs(args) {
             return reject(error);
         }, 30 * 60 * 1000);
 
-        logs.once('close', () => {
-            clearTimeout(timeout);
+        if(!args.raw) {
+            logs.once('close', () => {
```

Comment:
The problem here is that in `raw` mode, there isn't stream at all but just the console. Inside the Logs.createLogStream() the console is also treated as a stream, so it will break there too: https://github.com/auth0/wt-cli/blob/02bfb1ff88f0783c6f80580b629ef68697a75301/lib/logs.js#L26

The logs module needs to be updated to wrap the console in a stream, then this code would work as expected.